src/dex/quickswap/quickswap-e2e.test.ts

```diff
@@ -0,0 +1,357 @@
+            it('HANZO -> WMATIC', async () => {
+              await testE2E(
+                tokens.HANZO,
+                tokens.WMATIC,
+                holders.HANZO,
+                '41234567000000000',
+                SwapSide.SELL,
+                dexKey,
+                ContractMethod.megaSwap,
+                network,
+                provider,
+                undefined,
+                undefined,
+                { srcFee: 0, destFee: 0, srcDexFee: 500, destDexFee: 0 },
+              );
+            });
+          });
+          describe('swapOnUniswapV2Fork', () => {
+            it('WMATIC -> HANZO', async () => {
+              await testE2E(
+                tokens.WMATIC,
+                tokens.HANZO,
+                holders.WMATIC,
+                '1000000000000000000',
+                SwapSide.SELL,
+                dexKey,
+                ContractMethod.swapOnUniswapV2Fork,
+                network,
+                provider,
+                undefined,
+                undefined,
+                { srcFee: 0, destFee: 0, srcDexFee: 0, destDexFee: 500 },
+              );
+            });
+            it('HANZO -> WMATIC', async () => {
+              await testE2E(
+                tokens.HANZO,
+                tokens.WMATIC,
+                holders.HANZO,
+                '41234567000000000',
+                SwapSide.SELL,
+                dexKey,
+                ContractMethod.swapOnUniswapV2Fork,
+                network,
+                provider,
+                undefined,
+                undefined,
+                { srcFee: 0, destFee: 0, srcDexFee: 500, destDexFee: 0 },
+              );
+            });
+          });
+        });
+        describe('buy', () => {
+          describe('buy', () => {
+            it('HANZO -> WMATIC', async () => {
+              await testE2E(
+                tokens.HANZO,
+                tokens.WMATIC,
+                holders.HANZO,
+                '1000000000000000000',
+                SwapSide.BUY,
+                dexKey,
+                ContractMethod.buy,
+                network,
+                provider,
+                undefined,
+                undefined,
+                { srcFee: 0, destFee: 0, srcDexFee: 500, destDexFee: 0 },
+              );
+            });
+          });
+          describe('buyOnUniswapV2Fork', () => {
+            it('HANZO -> WMATIC', async () => {
+              await testE2E(
+                tokens.HANZO,
+                tokens.WMATIC,
+                holders.HANZO,
+                '1000000000000000000',
+                SwapSide.BUY,
+                dexKey,
+                ContractMethod.buyOnUniswapV2Fork,
+                network,
+                provider,
+                undefined,
+                undefined,
+                { srcFee: 0, destFee: 0, srcDexFee: 500, destDexFee: 0 },
+              );
+            });
+          });
+        });
+      });
+    });
+  });
+
+  describe('Arbitrum', () => {
```

Comment:
@paraschiv-razvan-toptal Not sure it'a good place to put this test case since the file name is `uniswap-v2-e2e-polygon...`. There is a separate file for Arbitrum tests https://github.com/paraswap/paraswap-dex-lib/blob/master/src/dex/uniswap-v2/uniswap-v2-e2e-arbitrum.test.ts